hail/src/main/scala/is/hail/io/RowStore.scala

```diff
@@ -149,6 +165,46 @@ final case class PackCodecSpec(child: BufferSpec) extends CodecSpec {
   def buildEncoder(t: PType): (OutputStream) => Encoder = { out: OutputStream =>
```

Comment:
I'd like to hook your code up to the encoder.  Right now it has a `writeRegionValue` that writes a single.  You could implement that, or you could implement a function that takes an RegionValueIterator and writes them all.  Or both!